src/Sylius/Behat/Context/Transform/ProductContext.php

```diff
@@ -41,14 +41,15 @@ public function __construct(ProductRepositoryInterface $productRepository)
      */
     public function getProductByName($productName)
     {
-        $product = $this->productRepository->findOneByName($productName);
+        $products = $this->productRepository->findByName($productName, 'en_US');
```

Comment:
For now we use English language in Behat I think, but it obviously should be changed (getting locale from shared storage?)